must-call-qual/src/main/java/org/checkerframework/checker/mustcall/qual/MustCall.java

```diff
@@ -0,0 +1,40 @@
+package org.checkerframework.checker.mustcall.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.DefaultFor;
+import org.checkerframework.framework.qual.DefaultQualifierInHierarchy;
+import org.checkerframework.framework.qual.SubtypeOf;
+import org.checkerframework.framework.qual.TypeUseLocation;
+
+/**
+ * If an expression has type {@code @MustCall({"m1", "m2"})}, then its value might have an
+ * obligation to eventually call methods {@code m1}, {@code m2}, both, or neither before any exit
+ * point or de-allocation. The value cannot have an obligation to call any methods that are not
+ * named in the annotation.
+ *
+ * <p>The subtyping relationship is:
+ *
+ * <pre>{@code @MustCall({"m1"}) <: @MustCall({"m1", "m2"})}</pre>
+ *
+ * <p>For example, an object with an obligation to call only "m1" can be passed to a method whose
+ * argument has an obligation to call both "m1" and "m2", because if both "m1" and "m2" are called,
```

Comment:
The "if both ..." text is different than the wording in [your comment](https://github.com/kelloggm/object-construction-checker/pull/207#discussion_r513117386).  Can you make them consistent?
My concern is that the comment about "if both m1 and m2 are called" does not follow, because a `@MustCall(m1, m2)` might have neither m1 nor m2 called on it.

As a general rule, it's usually better to just update the pull request, rather than writing an explanation in the pull request.  People reading the code won't have access to the pull request conversation, and the code should stand on its own.